test/full/tools/result.py

```diff
@@ -107,6 +107,6 @@ def results(self, verbose_level=1):
         return msg
 
     def succeeded(self):
-        if self.total == self.counts[ResultValue.PASS]:
-            return 0
-        return 1
+        if self.case_count == self.counts[ResultValue.PASS]:
```

Comment:
why not just return :P

Reply:
Yeah, used to be more complex in the past and got reduced to this. Will fix.